src/lvl_script.c

```diff
@@ -6976,6 +6996,8 @@ const struct CommandDesc command_desc[] = {
   {"SET_HEART_HEALTH",                  "PN      ", Cmd_SET_HEART_HEALTH, NULL, NULL},
   {"ADD_HEART_HEALTH",                  "PNN     ", Cmd_ADD_HEART_HEALTH, NULL, NULL},
   {"CREATURE_ENTRANCE_LEVEL",           "PN      ", Cmd_CREATURE_ENTRANCE_LEVEL, NULL, NULL},
+  {"RANDOMISE_FLAG",                    "PAN     ", Cmd_RANDOMISE_FLAG, NULL, NULL},
+  {"RANDOMIZE_FLAG",                    "PAN     ", Cmd_RANDOMISE_FLAG, NULL, NULL},
```

Comment:
The commands follow British english, so you may remove the 'randomize' one. Just one variation is enough.